.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "numpy/core/src/umath/svml"]
 	path = numpy/core/src/umath/svml
 	url = https://github.com/numpy/SVML.git
+[submodule "numpy/core/src/npysort/x86-simd-sort"]
+	path = numpy/core/src/npysort/x86-simd-sort
+	url = https://github.com/intel/x86-simd-sort
```

Comment:
The last hash [7d7591cf5927e83e4a1e7](https://github.com/intel/x86-simd-sort/commit/7d7591cf5927e83e4a1e7c4b6f2c4dc91a97889f) should be used instead to avoid any new changes that may break the build.

Reply:
It is pointing to the latest hash. 

```
>> git submodule
 36f4c898f2255e0c98eb6949cd67381552d5ffea doc/source/_static/scipy-mathjax (heads/master)
 7d7591cf5927e83e4a1e7c4b6f2c4dc91a97889f numpy/core/src/npysort/x86-simd-sort (remotes/origin/HEAD)
 1c5260a61e7dce6be48073dfa96291edb0a11d79 numpy/core/src/umath/svml (remotes/origin/HEAD)
```

Reply:
right, the `.gitmodules` file only holds the url, the actual commit [is stored elsewhere](https://devops.stackexchange.com/questions/5371/where-is-the-git-submodule-revision-actually-stored)